Cargo.toml

```diff
@@ -14,8 +14,9 @@ crate-type = ["cdylib"]
 pyo3-build-config = "0.19.1"
 
 [dependencies]
+base64 = "0.21"
```

Comment:
This is necessary to access the `base64` error that `tantivy` holds. This dependency won't change much otherwise, since it's also being used by `tantivy` anyways.